src/game_interpreter.cpp

```diff
@@ -3076,9 +3076,6 @@ bool Game_Interpreter::DefaultContinuation(RPG::EventCommand const& /* com */) {
 
 void Game_Interpreter::ResetEventCalling() {
 	event_calling = {};
-	// FIXME: Need to separate immediate battle calls from events
-	// and SavePartyLocation::encounter_calling for random encounters.
-	Game_Temp::battle_calling = false;
```

Comment:
I don't see any other change related to battle calling. Is just deleting this really the solution?